api/net/Areas/Services/Controllers/ContentController.cs

```diff
@@ -151,25 +170,39 @@ public async Task<IActionResult> AddAsync(ContentModel model)
     /// Publish message to kafka to index content in elasticsearch.
     /// </summary>
     /// <param name="model"></param>
+    /// <param name="index">Be careful this can result in a indexing loop.</param>
+    /// <param name="requestorId">The user ID who is requesting the update.</param>
     /// <returns></returns>
     [HttpPut("{id}")]
     [Produces(MediaTypeNames.Application.Json)]
     [ProducesResponseType(typeof(ContentModel), (int)HttpStatusCode.OK)]
     [ProducesResponseType(typeof(ErrorResponseModel), (int)HttpStatusCode.BadRequest)]
     [SwaggerOperation(Tags = new[] { "Content" })]
-    public async Task<IActionResult> UpdateAsync(ContentModel model)
+    public async Task<IActionResult> UpdateAsync(ContentModel model, bool index = false, int? requestorId = null)
```

Comment:
We don't want to cause an update/index loop when the Indexing Service updates content.